### ISSUE TITLE
Fieldset disabled no longer disables Select2

### ISSUE THREAD

**Issue by jeverden** [select2/select2#2030]
Field set disabled no longer disables select 2. This worked in the prior version 3.2.

**Comment by kevin-brown** [comment]
This should be fixed in 4.0.

If it is not fixed, please add a :+1: to the ticket to bring it to our attention. Please also include an example that we can test to verify that it is not actually working.

**Comment by luciusli** [comment]
Using jQuery 2.1.3 and Select2 4.0.0, it's not working for me on Chrome 45
http://jsfiddle.net/2fu4ws2y/

**Comment by kevin-brown** [comment]
Yup, looks like this is still an issue. We are going to need to figure out what check Sizzle used to use to determine the disabled status of an element. Clearly it's not `prop('disabled')`, as that returns false in Chrome.

**Comment by luciusli** [comment]
I believe `.is('.disabled')` is quite reliable.

**Comment by stief510** [comment]
Hey we just updated 4.0.0 and it still seems to be an issue. I was just wondering if there were any plans for an update with a fix. Thanks.

**Comment by cmcnulty** [comment]
It's a fundamental problem with Internet explorer, that said there is a jquery workaround in the works. 
On Feb 18, 2016 5:39 PM, Aryeh Stiefel <notifications@github.com> wrote:Hey we just updated 4.0.0 and it still seems to be an issue. I was just wondering if there were any plans for an update with a fix. Thanks.

**Comment by stief510** [comment]
For me it seems to not work even in Chrome

**Comment by nwj** [comment]
I can also confirm that disabling a fieldset fails to disable a select2 within the fieldset in Chrome on v4.0.0.

**Comment by phazei** [comment]
I'm experiencing this as well.
I did notice the fieldset disabled selector issue was recently fixed in jquery:
https://github.com/jquery/sizzle/pull/283
https://github.com/jquery/sizzle/issues/174

Seems it will roll out in JQ2.3.

**Comment by Karmalakas** [comment]
Any luck fixing the issue? It still is present

Chrome 54
Select2 4.0.3
jQuery 2.2.3

**Comment by phazei** [comment]
This issue is not stale.  It still exists in 4.0.12
Also, IE doesn't matter much anymore.  All the other browsers support disabled fieldset including Edge:
https://caniuse.com/#feat=fieldset-disabled

Please re-open.

**Comment by debnet** [comment]
I faced the issue today when using fieldsets in conjunction with select2 fields, all fields excluding select2 selections was greyed out.
Please re-open.

**Comment by fire-birdie** [comment]
I ran into this issue yesterday and woke up early to hack together a fix for it. Unfortunately, I'm not familiar with the unit test framework used by the project, and I don't have time to figure it out right now - I'll try to get to it this weekend. I'll open a PR shortly in case anybody wants to review the changes in the meantime, or can assist with the tests.

**Comment by Herz3h** [comment]
Any news on this ?

**Comment by ivke013** [comment]
Any news?

**Comment by Skarkolainen** [comment]
Any news ?

**Comment by cgeorg** [comment]
Bump, this seems like it should be basic supported functionality, especially if the select2-ized select is already in a disabled fieldset when it's initialized.
